views/sessions/new.html

```diff
@@ -13,3 +13,7 @@
   </div>
   <input type="submit" value="Submit" class="form__submit-button"/>
 </form>
+
+<span class="small">
```

Comment:
There is actually a tag for this (see [reference](https://www.w3schools.com/tags/tag_small.asp)):

```suggestion
<small>
```

Reply:
@olivierobert ✅  fixed in [a945541](https://github.com/carryall/go-google-scraper-challenge/pull/32/commits/a945541b662b58c1f0c51ca0e2826edf7e167beb)